tests/lean/run/bv_axiom_check.lean

```diff
@@ -2,7 +2,7 @@ import Std.Tactic.BVDecide
 
 open BitVec
 
-theorem bv_axiomCheck (x y : BitVec 1) : x + y = y + x := by
+theorem bv_axiomCheck (x : BitVec 2) : x &&& 1 = (x <<< 1) >>> 1 := by
```

Comment:
Just reporting here because I was a little confused at first: this didn't fail until now because `add_eq_xor` rewrote (specifically 1-bit) addition into a xor before the `add_*_inj` theorems applied. Not sure if we'd like to mark that lemma low in the simpset, or how to properly ensure confluence in cases like this.

Reply:
I see, could we potentially change this to just proving transitivity of `<` on `BitVec`? That's something that our rewrite rules can't do currently so this should hopefully always be safe from being solved by basic rewrites.